### ISSUE TITLE
[Change Request]: additional template vehicle categories 

### ISSUE THREAD

**Issue by ante185** [official-antistasi-community/A3-Antistasi#2870]
### Is the requested change related to a problem?

Currently there is no distinction in the templates between for example jeeps/softskinned and hardier MRAPs the latter of which may be desirable to make breachable.
Unarmed APCs and other armoured-but-unarmed transports really don't have a category that they can go in either as the APC categories do assume that they are armed, and using them in the truck category won't have them be breachable vehicles.

a category for vehicles which are neither tanks or APC/IFV (transports), but too heavy to be LightArmed would be an idea to ponder as well.

### Describe the solution you'd like.

Additional vehicle categories:
armed/unarmed MRAP/lightArmoured - vehicles which are buttoned down and locked up, better armoured and requires breaching (Could have easier requirements for breaching than APCs)
armoured troop transport - unarmed APCs and armoured trucks for use at later war levels

Additionally some staging categories for tanks, ie light and now that spearhead is out we might want a heavy-tank category while we're at it?

### Alternative solutions

It has been proposed that the breachabilty of vehicles could be put into the vehicle attributes - and while that would probably suffice for the number of applicable vehicles we could think off, it is also kind of a short term idea - splitting them up by category would be a more solid and long term solution.

**Comment by Tiny-DM** [comment]
With the introduction of lightAPCs, lightTanks and heavyTanks (and IFVs maybe? I dont remember if that's new) we've achieved a good equilibrium. It's still an issue that IFVs require the ability to transport troops IMO, but that's more of a structural issue that might involve putting IFVs in the APC category like we usually do. Closing.

**Comment by ante185** [comment]
> It's still an issue that IFVs require the ability to transport troops IMO, but that's more of a structural issue that might involve putting IFVs in the APC category like we usually do. Closing.

non-troop carrying IFVs can go in the light tank category, and lightArmed could, should the situation ever arise be used for non-troop carrying light APCs. really it's only non-troop carrying "APC"s (Like the NATO utility APC) that do not have any place.
In general this issue is targeting the battle busses both RHS and CUP provide, and the general trend for a lot of factions to not actually having MRAPs and instead they have humvees, jeeps, and technicals.
Which is something i don't think we're addressing well yet.
